src/styles/blocks/card.scss

```diff
@@ -0,0 +1,109 @@
+.card {
+  margin: 0;
+  font-family: Roboto, serif;
+  text-align: left;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  width: 200px;
+
+  border: 1px #f3f3f3 solid;
+
+  &__image {
+    height: 134px;
+    width: 160px;
+    padding-bottom: 40px;
+    align-self: center;
+  }
+
+  &__title {
+    margin: 0;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    padding-bottom: 4px;
+    color: $main-color;
+  }
+
+  &__code {
+    margin: 0;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    padding-bottom: 16px;
+    color: $secondary-color;
```

Comment:
The variable `$secondary-color` is used here but it is not defined in the provided SCSS files. Ensure this variable is defined in your `variables.scss` file or another appropriate location.